### PR TITLE
3257 cleanup old auth part 6 tokenvalidation cleanup

### PR DESCRIPTION
**Pull request by sahma19**
This is part 6 of the following ZenHub issue: [https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/3257](https://app.zenhub.com/workspaces/team-atlas-633199659e255a37cd1d144f/issues/gh/energinet-datahub/energy-origin-issues/3257)

### CODE REVIEW

File: domains/libraries/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/B2C/TermsAcceptedRequirementTest.cs [hunk 1]
```diff
@@ -1,4 +1,6 @@
+using System.Collections.Generic;
 using System.Security.Claims;
+using System.Threading.Tasks;
```
Comment (TopSwagCode) [inline domains/libraries/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/B2C/TermsAcceptedRequirementTest.cs:3]: Why are these added?

File: domains/libraries/EnergyOrigin.TokenValidation/configuration.yaml [hunk 2]
```diff
@@ -1 +1 @@
-version: 3.6.8
```
Comment (TopSwagCode) [inline domains/libraries/EnergyOrigin.TokenValidation/configuration.yaml:1]: We don't use nuget anymore. So we shouldn't have to bump these anymore.

File: domains/libraries/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Utilities/TokenSignerTests.cs [hunk 3]
```diff
@@ -1,5 +1,8 @@
+using System;
+using System.Collections.Generic;
 using EnergyOrigin.TokenValidation.Utilities;
 using System.IdentityModel.Tokens.Jwt;
+using System.Linq;
```
[the remark this answers is not in the document]
Comment (sahma19) [reply domains/libraries/EnergyOrigin.TokenValidation/EnergyOrigin.TokenValidation.Unit.Tests/Utilities/TokenSignerTests.cs:5]: I actually don't know. but rider was complaining about missing libraries, while i was deleting files from tokenvalidation.

File: domains/libraries/EnergyOrigin.TokenValidation/configuration.yaml (hunk 2 above)
Comment (sahma19) [reply domains/libraries/EnergyOrigin.TokenValidation/configuration.yaml]: Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/1492/commits/1e019e674f303254bfcb614614c8c7c494b18e35

**Review by TopSwagCode — APPROVED** [review]
